examples/matching-engine/README.md

```diff
@@ -138,13 +146,121 @@ mutation ExecuteOrder {
 To query about the bid price:
 
 ```gql,uri=http://localhost:8080/chains/$CHAIN_1/applications/$MATCHING_ENGINE
-query{
+query {
   bids {
-    keys{
+    keys {
       price
     }
   }
 }
 ```
 
+
+### Atomic Swaps
+
+If you send tokens to a chain owned by someone else, you rely on them for liveness:
+If they don't handle your messages, you don't have access to your tokens.
+
+If the number of parties who want to swap tokens is limited, this can be addressed
+by making them all chain owners, allowing only matching engine operations on the chain,
+and allowing only the matching engine to close the chain.
+
+```bash
+kill %%    # Kill the service so we can use CLI commands for chain 1.
+
+linera --wait-for-outgoing-messages change-ownership \
```

Comment:
We could set an environment variable `LINERA_WAIT_FOR_OUTGOING_MESSAGES` (in another PR)

Reply:
That's not much shorter, though. :smile:
We could call it `LINERA`? The outgoing messages option is mostly important for the test anyway, I think?